include/zephyr/arch/arc/v2/linker.ld

```diff
@@ -112,23 +116,24 @@ SECTIONS {
 		KEEP(*(.ctors*))
 		_ectors = .;
 #endif /* CONFIG_CPLUSPLUS && !CONFIG_CPP_STATIC_INIT_GNU && __MWDT_LINKER_CMD__ */
-	} GROUP_LINK_IN(ROMABLE_REGION)
 
-#ifdef __MWDT_LINKER_CMD__
-/* TODO: add mwdt specific ROM C++ sections */
-#else
-#include <zephyr/linker/cplusplus-rom.ld>
-#endif /* __MWDT_LINKER_CMD__ */
+		MPU_ALIGN(ABSOLUTE(.) - __rom_region_start);
+	} GROUP_LINK_IN(ROMABLE_REGION)
 
 	__rodata_region_end = .;
-	MPU_ALIGN(__rodata_region_end - __rom_region_start);
+
+	 MPU_ALIGN(__rodata_region_end - __rom_region_start);
```

Comment:
Alignment issue - let's drop this extra space.

If the newline isn't intended we can drop it either.